cmd/security-secrets-setup/Dockerfile

```diff
@@ -66,15 +71,20 @@ COPY --from=build-env /edgex-go/cmd/security-secrets-setup/res/pkisetup-vault.js
 # Kong PKI/TLS materials
 COPY --from=build-env /edgex-go/cmd/security-secrets-setup/res/pkisetup-kong.json ./res
 
+# setup the entry point script
+# it uses `dumb-init` as the top-level process to reap any
+# zombie processes created by sub-processes
+COPY --from=build-env /edgex-go/cmd/security-secrets-setup/entrypoint.sh /usr/local/bin/
+RUN chmod +x /usr/local/bin/entrypoint.sh
+RUN ln -s usr/local/bin/entrypoint.sh /
+
 # Create assets folder (needed for unseal key/s, root token and tmp)
 # Run CA/Vault and Kong PKI/TLS setups and peform housekeeping tasks
 RUN mkdir /vault/config/assets && \
     chown -R vault:vault /vault && \
     chmod 644 /vault/config/local.hcl && \
-    chmod 744 security-secrets-setup && \
-    /vault/security-secrets-setup --config /vault/res/pkisetup-vault.json && \
-    echo "" && \
-    /vault/security-secrets-setup --config /vault/res/pkisetup-kong.json && \
-    chown -R vault:vault /vault/config/pki
+    chmod 744 security-secrets-setup
```

Comment:
Granted that this is not currently broken since root is running the executable, but executables should generally be chmod 755.

Reply:
744 was there originally. I agree that an executable file usually is 755.

Reply:
so I guess, the original intention is that this executable is owned by `vault` user as the ownership is transferred to `vault` user and `vault` group in line 71 of `docker-entrypoint.sh`.  So I'll leave as it is for now unless there is a strong objection.